borg/cache.py

```diff
@@ -140,10 +140,7 @@ def create(self):
         with open(os.path.join(self.path, 'files'), 'wb') as fd:
             pass  # empty file
 
-    def _do_open(self):
-        self.config = configparser.ConfigParser(interpolation=None)
-        config_path = os.path.join(self.path, 'config')
-        self.config.read(config_path)
```

Comment:
a better (minimal) diff tool for github would be really cool. :-|
the same lines are "added" below again.